### PR TITLE
Querry and update table

### PR DESCRIPTION
**Pull request by oguntayomicheal**
### The following were implemented in the PR.

- Insert more data
01 - Insert more data

- Update species to unspecified
02 - Update species to unspecified

- Rollback
03 - roll back

- Set species with a name ending in mon to Digmon
04 - species to digmon

- Set others to pokmon
05 - pokmon

- Commit transactions
06 - commit transactions

- Delete and rollback data
07 - delete and rollback datat

- delete and create a savepoint
08 - del and savepoint

- Update weight to -1
09 - update weight to -1

- Rollback and commit

10 -rollback and commit

- Queries for animals that are there and also, for those that have never tried to escape
11 - Animals that are there and those that have not escaped

- Average weight
12 - Average weight

- Animals that escaped the most
13 - Escaped the most

- Min and max weight of each type of animals
14 - Min and Max weight

- The average number of escape attempts per animal type of those born between 1990 and 2000
15 - Average of escape attempts

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -14,4 +14,42 @@ SELECT * FROM animals WHERE neutered = true;
...
-SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
\ No newline at end of file
+SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
+
+BEGIN;
+
+  UPDATE animals
+  SET species = 'unspecified';
+
+ROLLBACK;
```
Comment (bobb-Rob) [inline queries.sql:24]: - [ ] You are expected to have `SELECT species from animals;` query to verify that change was made before rollback and after rollback. E.g 👇🏾 

```sql
BEGIN; -- start transaction

   UPDATE animals
   SET species = 'unspecified'; -- make change

   SELECT species from animals; -- verify that change was made
   ROLLBACK;

   SELECT species from animals; -- verify that change was undone
```

File: queries.sql [hunk 2]
```diff
@@ -14,4 +14,42 @@ SELECT * FROM animals WHERE neutered = true;
...
+
+BEGIN;
+UPDATE animals SET species = 'digimon' WHERE name LIKE '%mon';
+SELECT * FROM animals;
+UPDATE animals SET species = 'Pokemon' WHERE species is NULL;
```
Comment (bobb-Rob) [inline queries.sql:29]: - [ ] Here, the line 28 code should have come before line 29.

File: queries.sql [hunk 3]
```diff
@@ -14,4 +14,42 @@ SELECT * FROM animals WHERE neutered = true;
...
+COMMIT;
+SELECT * FROM animals;
+
+BEGIN;
+DELETE FROM animals; 
+ROLLBACK;
+SELECT * FROM animals;
```
Comment (bobb-Rob) [inline queries.sql:36]: - [ ] After the delete on line 34, as you did in the screenshot, there should be a query following it to verify if they were actually deleted. Please add the query as stated in the project requirement. So you should have a query to verify before and after rollback.

File: queries.sql [hunk 4]
```diff
@@ -14,4 +14,42 @@ SELECT * FROM animals WHERE neutered = true;
...
+
+BEGIN;
+DELETE FROM animals WHERE date_of_birth > '2022-01-01';
+SAVEPOINT date_birth_Jan_1st_2022;
+UPDATE animals SET weight_kg = weight_kg * -1;
+ROLLBACK TO date_birth_Jan_1st_2022;
+UPDATE animals SET weight_kg = weight_kg * -1 WHERE weight_kg < 0;
```
Comment (bobb-Rob) [inline queries.sql:43]: - [ ] As per the requirement, after line 43 code, there should follow a `COMMIT`. I see that on your screenshots, the commit was performed. Please include the COMMIT in this file.

**Review by bobb-Rob — APPROVED** [review]
## STATUS: APPROVED ✔️ 🍾 🍾 🍾 🥳 🥳 🥳 

Hi @oguntayomicheal, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
✅ Correct use of Gitflow
✅ Very descriptive Pull Request(PR)

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: queries.sql (hunk 1 above)
Comment (oguntayomicheal) [reply queries.sql:24]: Hi @bobb-Rob, thanks for your optional suggestions. They will be implemented.

**Pull request merged**